jerry-core/ecma/builtin-objects/ecma-builtin-arraybuffer.c

```diff
@@ -98,11 +113,44 @@ ecma_builtin_arraybuffer_dispatch_construct (const ecma_value_t *arguments_list_
  * @return ecma_value
  *         returned value must be freed with ecma_free_value
  */
+/*
 ecma_value_t
-ecma_builtin_arraybuffer_species_get (ecma_value_t this_value) /**< This Value */
+ecma_builtin_arraybuffer_species_get (ecma_value_t this_value) < This Value
```

Comment:
Do we need this commented method or with this pull request it's become obsolete?